dev/com.ibm.ws.jca/src/com/ibm/ws/jca/internal/JCAWorkContext.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jca.internal;
+
+import java.io.Serializable;
+import java.util.HashMap;
+
+import com.ibm.wsspi.threading.WorkContext;
+
+//Implements WorkContext class to add context to JCA threads.
+public class JCAWorkContext extends HashMap<String, Serializable> implements WorkContext {
```

Comment:
The name of this class is very confusing, given that the JCA spec already defines a WorkContext of its own.  One would logically expect a class named JCAWorkContext to be a JCA WorkContext implementation.  But it is not.  Can we name it something else.  Maybe LibertyThreadingContext ?  Or at least include a comment calling out that it is not an implementation of javax.resource.spi.work.WorkContext.